cmds/info.c

```diff
@@ -281,6 +281,9 @@ static int read_cpuinfo(void *arg)
 			else if (!strncmp(info->cpudesc, "ARM64", 5)) {
 				handle->arch = UFT_CPU_AARCH64;
 			}
+			else if (!strncmp(info->cpudesc, "RISCV64", 7)) {
+				handle->arch = UFT_CPU_RISCV64;
+			}
```

Comment:
Looks like this part should be in #1815 .

Reply:
It's related to the `cpuinfo.c` code I added in #1815, so I'll move it as per your feedback.

Reply:
Move this to https://github.com/namhyung/uftrace/pull/1815 based on feedback, but also move utils/arch.h to https://github.com/namhyung/uftrace/pull/1815.
because UFT_CPU_RISCV64 was declared in utils/arch.h in commit 1a4d0ec.